app/src/main/java/com/example/habitsmasher/PublicPrivateButtons.java

```diff
@@ -0,0 +1,131 @@
+package com.example.habitsmasher;
+
+import static android.content.ContentValues.TAG;
+
+import android.nfc.Tag;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CheckBox;
+
+import com.google.android.material.tabs.TabLayout;
+
+public class PublicPrivateButtons {
+    /**
+     * This class handles the logic behind the public and private buttons, which are used to
+     * set a habit to either public or private.
+     *
+     * @author Cameron Matthew
+     * @version 1.0
+     */
+
+    private Button _publicButton;
+    private Button _privateButton;
+    private boolean _publicSelected;
+    private boolean _privateSelected;
```

Comment:
If privateSelected and publicSelected are complements of each other, wouldn't it just been be easier to have one boolean variable controlling the logic

Reply:
True, but for code clarity I thought it was worth the complexity.

Reply:
I think in this case I'd agree that one variable would make more sense